### ISSUE TITLE
confused about Figure 4 in the paper

### ISSUE THREAD

**Issue by fanjing8** [UniBwTAS/YOLOPoint#4]
Congratulations on your great job!

![image](https://github.com/UniBwTAS/YOLOPoint/assets/26003725/47ffc35a-9a7c-4a64-b5ab-61e50f8b8f44)

I am a little confused about Figure 4 in the paper. From the meaning of the Figure, we can know that the method in the lower left corner is better. But for the four methods YPL, YPM, YPS, and YPN, why do the translation errors and rotation errors increase as the model capacity increases? For example, the translation error and rotation error of YPL are larger than those of the YPM method.

Looking forward to your reply, thank you.

**Comment by antopost** [comment]
Hi VEDFU! Great observation. The short answer is: I'm not sure and can only speculate (which is why I didn't address it in the paper)
In the case of YPL I had trouble fine-tuning it on KITTI. After a certain number of epochs, the gradient would just explode and the accuracy would go to 0. I tried gradient clipping and freezing some parts of the net, but the gradient would eventually explode, which is why I stopped the training prematurely, even though there was still some potential.

As for why YPN is more accurate than YPS in this experiment. I really couldn't say.
On the one hand, the errors are so close that it could just be statistical noise.
Another explanation could be that matching accuracy does not scale well with descriptor vector length. For example, ORB's descriptor is only 32 and it works fine. In other words, keypoints don't necessarily benefit from having longer descriptor vectors.
